### PR TITLE
chore: fix eslint-plugin-eslint-comments not found

### PR DESCRIPTION
**Pull request by geekrainy**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Develop env optimization

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by xrkffgg** [comment]
可以了解一下 commit 规范

File: package.json [hunk 1]
```diff
@@ -201,6 +201,7 @@
     "eslint-config-airbnb": "^18.0.0",
     "eslint-config-prettier": "^6.0.0",
     "eslint-plugin-babel": "^5.3.0",
+    "eslint-plugin-eslint-comments": "^3.2.0",
```
Comment (xrkffgg) [inline package.json:204]: 为啥加这个？

Comment (geekrainy) [reply package.json:204]: 不加，开发环境跑不起来

**Comment by afc163** [comment]
我本地实测是没问题的，给一下重现步骤。

**Comment by xrkffgg** [comment]
这个应该是没有问题的，检查下自己的环境。

**Comment by geekrainy** [comment]
yarn 缓存导致依赖没装到，关掉了。

**Pull request closed without merging**